src/task/SubscribeAndReceiveSubscribeAckTask.ts

```diff
@@ -88,14 +106,25 @@ export default class SubscribeAndReceiveSubscribeAckTask extends BaseTask {
       isSendingStreams,
       this.context.videoStreamIndex.localStreamDescriptions(),
       // TODO: handle check-in mode, or remove this param
-      true
+      true,
+      compressedSDPOffer
     );
     this.context.logger.info(`sending subscribe: ${JSON.stringify(subscribe)}`);
     this.context.signalingClient.subscribe(subscribe);
 
     const subscribeAckFrame = await this.receiveSubscribeAck();
     this.context.logger.info(`got subscribe ack: ${JSON.stringify(subscribeAckFrame)}`);
-    this.context.sdpAnswer = subscribeAckFrame.sdpAnswer;
+    let decompressedText: string = '';
```

Comment:
Can drop `: string` and due to `''` TypeScript will assign the `string` type.

Reply:
Thanks. will do